lib/formatter.ex

```diff
@@ -29,6 +34,7 @@ defmodule TelemetryInfluxDB.Formatter do
   defp to_bin(val) when is_integer(val), do: Integer.to_string(val)
   defp to_bin(val) when is_float(val), do: Float.to_string(val)
   defp to_bin(val) when is_atom(val), do: Atom.to_string(val)
+  defp to_bin(%DateTime{} = val), do: DateTime.to_unix(val, :nanosecond) |> Integer.to_string()
```

Comment:
Does doing this conversion here change other behavior in the library? We don't currently allow `DateTime`s to be tags and measurements, but this will make that legal and I'm not sure it should. It might be better to inline this conversion into `format` above instead, or make a different private helper function.

Reply:
Ah, you're right. I will change it.